internal/commands/inspect.go

```diff
@@ -0,0 +1,102 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/deislabs/cnab-go/action"
+	"github.com/docker/app/internal"
+	"github.com/docker/app/internal/cnab"
+	"github.com/docker/app/internal/inspect"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+type inspectOptions struct {
+	credentialOptions
+	pretty       bool
+	orchestrator string
+}
+
+func inspectCmd(dockerCli command.Cli) *cobra.Command {
+	var opts inspectOptions
+	cmd := &cobra.Command{
+		Use:   "inspect [APP_NAME] [OPTIONS]",
+		Short: "Shows installation and application metadata, parameters and the containers list of a running application",
+		Example: `$ docker app inspect my-installed-app
+$ docker app inspect my-app:1.0.0`,
```

Comment:
I wonder if that example is correct, shouldn't it be `docker app image inspect` ?

Reply:
image inspect command has been move to `commands/image` package